spec/system/agenda_spec.rb

```diff
@@ -0,0 +1,143 @@
+require "rails_helper"
+
+RSpec.describe "Agenda", type: :system do
+  let!(:user) { create(:user) }
+  let(:conference) { create(:conference) }
+  let!(:session) { create(:session, :live, :with_speakers, conference: conference) }
+  let(:speaker) { session.speakers.first }
+
+  before do
+    Timecop.freeze(Time.current.change(hour: 12))
```

Comment:
Because sessions are based on a datetimes, we need to freeze time to avoid random failures